src/platforms/macos/daemon/dnsutilsmacos.h

```diff
@@ -25,27 +22,11 @@ class DnsUtilsMacos final : public DnsUtils {
                        const QList<QHostAddress>& resolvers) override;
   bool restoreResolvers() override;
 
- private:
-  void backupResolvers();
-  void backupService(const QString& uuid);
+ private slots:
+  void dnsManagerStdoutReady();
 
  private:
-  class DnsBackup {
-   public:
-    DnsBackup() {}
-    bool isValid() const {
-      return !m_domain.isEmpty() || !m_search.isEmpty() ||
-             !m_servers.isEmpty() || !m_sortlist.isEmpty();
-    }
-
-    QString m_domain;
-    QStringList m_search;
-    QStringList m_servers;
-    QStringList m_sortlist;
-  };
-
-  SCDynamicStoreRef m_scStore = nullptr;
-  QMap<QString, DnsBackup> m_prevServices;
+  QProcess m_dnsManager;
```

Comment:
This could use a comment that it will be used to create a watch dog process for DNS cleanup.

Reply:
Sure, I will rename it.